src/it/java/com/google/checkstyle/test/base/BaseCheckTestSupport.java

```diff
@@ -82,7 +82,7 @@ protected static Configuration getConfiguration() throws CheckstyleException {
      * @param clazz check class.
      * @return {@link DefaultConfiguration} instance.
      */
-    protected static DefaultConfiguration createCheckConfig(Class<?> clazz) {
+    private static DefaultConfiguration createCheckConfig(Class<?> clazz) {
```

Comment:
Can be moved closer to https://github.com/checkstyle/checkstyle/blob/dd178c43a1bc6bee90081db7aa595cb0c51974b7/src/it/java/com/google/checkstyle/test/base/BaseCheckTestSupport.java#L116